appengine/pond/js/blocks.js

```diff
@@ -246,568 +247,3 @@ Blockly.JavaScript['pond_loc_y'] = function(block) {
-      this.appendValueInput('IF' + i)
-          .setCheck('Boolean')
-          .appendField('} else if (');
-      this.appendDummyInput('TAIL' + i)
-          .appendField(') {');
-      this.appendStatementInput('DO' + i);
-    }
-    if (this.elseCount_) {
-      this.appendDummyInput('ELSEMSG')
-          .appendField('} else {');
-      this.appendStatementInput('ELSE');
-    }
-    // Move final '}' to the end.
-    this.moveInputBefore('TAIL', null);
-  },
-  mutationToDom: Blockly.Blocks['controls_if'].mutationToDom,
-  domToMutation: Blockly.Blocks['controls_if'].domToMutation,
-  decompose: Blockly.Blocks['controls_if'].decompose,
-  compose: Blockly.Blocks['controls_if'].compose,
-  saveConnections: Blockly.Blocks['controls_if'].saveConnections
-};
-
-Blockly.JavaScript['pond_controls_if'] =
-    Blockly.JavaScript['controls_if'];
-
-/**
- * Comparison operator.
- * @this Blockly.Block
- */
-Blockly.Blocks['logic_compare'].init = function() {
-  var OPERATORS = [
-    ['==', 'EQ'],
-    ['!=', 'NEQ'],
-    ['<', 'LT'],
-    ['<=', 'LTE'],
-    ['>', 'GT'],
-    ['>=', 'GTE']
-  ];
-  this.setHelpUrl(Blockly.Msg.LOGIC_COMPARE_HELPURL);
-  this.setColour(Blockly.Blocks.logic.HUE);
-  this.setOutput(true, 'Boolean');
-  this.appendValueInput('A');
-  this.appendValueInput('B')
-      .appendField(new Blockly.FieldDropdown(OPERATORS), 'OP');
-  this.setInputsInline(true);
-  // Assign 'this' to a variable for use in the tooltip closure below.
-  var thisBlock = this;
-  this.setTooltip(function() {
-    var op = thisBlock.getFieldValue('OP');
-    var TOOLTIPS = {
-      EQ: Blockly.Msg.LOGIC_COMPARE_TOOLTIP_EQ,
-      NEQ: Blockly.Msg.LOGIC_COMPARE_TOOLTIP_NEQ,
-      LT: Blockly.Msg.LOGIC_COMPARE_TOOLTIP_LT,
-      LTE: Blockly.Msg.LOGIC_COMPARE_TOOLTIP_LTE,
-      GT: Blockly.Msg.LOGIC_COMPARE_TOOLTIP_GT,
-      GTE: Blockly.Msg.LOGIC_COMPARE_TOOLTIP_GTE
-    };
-    return TOOLTIPS[op];
-  });
-  this.prevBlocks_ = [null, null];
-};
-
-Blockly.Msg.LOGIC_OPERATION_AND = '&&';
-Blockly.Msg.LOGIC_OPERATION_OR = '||';
-
-Blockly.Msg.LOGIC_BOOLEAN_TRUE = 'true';
-Blockly.Msg.LOGIC_BOOLEAN_FALSE = 'false';
-
-Blockly.Blocks['pond_loops_while'] = {
-  /**
-   * Do while/until loop.
-   * @this Blockly.Block
-   */
-  init: function() {
-    this.jsonInit({
-      'message0': 'while ( %1 ) { %2 %3 }',
-      'args0': [
-        {
-          'type': 'input_value',
-          'name': 'BOOL',
-          'check': 'Boolean'
-        },
-        {
-          'type': 'input_dummy'
-        },
-        {
-          'type': 'input_statement',
-          'name': 'DO'
-        }
-      ],
-      'previousStatement': null,
-      'nextStatement': null,
-      'colour': 120,
-      'tooltip': Blockly.Msg.CONTROLS_WHILEUNTIL_TOOLTIP_WHILE,
-      'helpUrl': Blockly.Msg.CONTROLS_WHILEUNTIL_HELPURL
-    });
-  }
-};
-
-Blockly.JavaScript['pond_loops_while'] =
-    Blockly.JavaScript['controls_whileUntil'];
-
-Blockly.Blocks['pond_math_number'] = {
```

Comment:
This one can be restored to Pond, since Genetics would have no need for the custom extension here which is to automatically switch to degrees for angle inputs.